huxley/www/js/components/RegistrationView.js

```diff
@@ -5,6 +5,9 @@
 
 'use strict';
 
+// Only necessary until ES2017 becomes standard
```

Comment:
This should go in the entry file (`www/js/entry.js`). Notice that it already has a corejs polyfill in there (we don't want to require the entire polyfill to keep our JS size down). Determine which polyfill we need, perhaps it's just `core-js/es7`, and put that there.